### ISSUE TITLE
Have the option to choose min-width

### ISSUE THREAD

**Issue by HoussemNedjar** [nuxt/image#1211]
I've been using Nuxt Image and have noticed that it primarily uses max-width for image generation. I'm interested in discussing the possibility of having an option to choose min-width instead, as it can be beneficial in certain scenarios.

**Comment by michaelpumo** [comment]
Yeah, I would like to see an option that can flip the media queries.
